docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -0,0 +1,196 @@
+---
+redirect_from:
+- "/docs/user-guide/persistent-volumes/walkthrough/"
+- "/docs/user-guide/persistent-volumes/walkthrough.html"
+title: Configuring a Pod to Use a PersistentVolume for Storage
+---
+
+{% capture overview %}
+
+This page shows how to configure a Pod to use a PersistentVolume for stoarage.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+You need to have a Kubernetes cluster that has only one Node, and the kubectl
+command-line tool must be configured to communicate with your cluster. If you
+do not already have a single-node cluster, you can create one by using
+[Minikube](/docs/getting-started-guides/minikube).
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Creating an index.html file on your Node
+
+Open a shell to the Node in your cluster. How you open a shell depends on how
+you set up your cluster. For example, if you are using Minikube, you can open a
+shell to your Node by entering `minikube ssh`.
+
+In your shell, create a `/tmp/data` directory:
+
+    mkdir /tmp/data
+
+In the `/tmp/data` directory, create an `index.html` file:
+
+    echo 'Hello from Kubernetes storage' > /tmp/data/index.html
+
+## Creating a PersistentVolume
```

Comment:
I forgot how complicated this all is.  Does it make sense to quickly provide an overview of the moving pieces and their relationship with one another?  My first read through this, it wasn't clear to me why all these things existed.

Roughly:
- PersistentVolume: Unallocated persistent storage provisioned by cluster admins that can be allocated.
- PersistentVolumeClaim: Allocation of persistent storage defined by a PersistentVolume.  Can be referenced within a Pod to mount storage as a Volume.
- Pod.Volume: Binds pre-allocated persistent storage for use within the Pod.